Plugins/BrickGrid/Source/BrickGrid/Private/BrickRenderComponent.cpp

```diff
@@ -328,7 +336,23 @@ UBrickRenderComponent::UBrickRenderComponent( const FObjectInitializer& Initiali
 FPrimitiveSceneProxy* UBrickRenderComponent::CreateSceneProxy()
 {
 	const double StartTime = FPlatformTime::Seconds();
-
+	TArray<EBrickClass> BrickClassByMaterial;
+	for (int32 iterator = 0; iterator < Grid->Parameters.Materials.Num(); ++iterator)
+	{
+		if (iterator == Grid->Parameters.EmptyMaterialIndex)
+			BrickClassByMaterial.Add(EBrickClass::Empty);
```

Comment:
To match the UE4 coding style here, please name `iterator` something like `MaterialIndex`, and add curly braces to this single statement `if`.